### ISSUE TITLE
Special Char fix not working in latest master

### ISSUE THREAD

**Issue by KirubakaranSP** [hugegraph/hugegraph#1688]
### Bug Type (问题类型)

other exception / error (其他异常报错)

### Before submit

- [X] 我已经确认现有的 [Issues](https://github.com/hugegraph/hugegraph/issues) 与 [FAQ](https://hugegraph.github.io/hugegraph-doc/guides/faq.html) 中没有相同 / 重复问题

### Environment (环境信息)

- Server Version: v11.5.2
- Backend: RocksDB 
- OS: macOS Big Sur CPUs, xx G RAM, Centos 7.x 
- Data Size:  xx vertices, xx edges 

### Expected & Actual behavior (期望与实际表现)

Already the special char values in search indexed property has been reported and a fix was pushed to master.Kindly check [here](https://github.com/hugegraph/hugegraph/issues/1638).But still the problem exists and throws new excpetion while inserting the same values.

Exception : 
{"exception":"class java.lang.IllegalArgumentException","message":"The SEARCH_INDEX type index id can't contains byte '0x00', but got: 0x503a78797a00616263","cause":""}

**Comment by javeme** [comment]
@KirubakaranSP Yes, the master branch still restricts special characters in search indexed property, and you can modify the source code if you have special needs. Please refer to: https://github.com/hugegraph/hugegraph/issues/1638#issuecomment-992570312
